main.py

```diff
@@ -240,8 +240,11 @@ def realtime():
 @app.route('/realtime/dummy', methods=["GET"])
 def dummy_data():
     test_sensors = \
-        {sensor: round(uniform(float(db_format[sensor]["safe_min"]), float(db_format[sensor]["safe_max"])), 4) for category in rt_format.keys() for sensor in rt_format[category]["lines"]}
+        {sensor: round(uniform(float(db_format[sensor]["safe_min"]*0.8), float(db_format[sensor]["safe_max"]*1.2)))for category in rt_format.keys() for sensor in rt_format[category]["lines"]}
 
+    test_sensors["bms_fault"] = randint(1, 1000000)
+    test_sensors["solar_fault"] = randint(1, 1000000)
+    test_sensors["motor_fault"] = randint(1, 1000000)
```

Comment:
A more realistic way to test this could be to have a small (e.g. 5%) chance of each bit being 1